app/models/search_builder.rb

```diff
@@ -12,4 +15,16 @@ class SearchBuilder < Blacklight::SearchBuilder
   #   def add_custom_data_to_query(solr_parameters)
   #     solr_parameters[:custom] = blacklight_params[:user_value]
   #   end
+
+  self.default_processor_chain += [:min_match_for_boolean]
+
+  # Sets min match to 1 based on presence of boolean operators, overriding default set in Solr
```

Comment:
```suggestion
  # Sets `mm` (minimum should match) to 1 based on presence of boolean operators, overriding default set in Solr.
  # The default value in the request handler (`6<-1 6<90%`) allows for some slop in full-text queries,
  # but leads to unexpected behavior when a user expects to use boolean operators for more advanced
  # search queries. Setting `mm=1` means only 1 clause needs to match, but the lucene query parser (in 
  # edismax) will precompose the query strings so everything works out.
```

Is that a fair summary of why mm=1?

Reply:
yes, I think 0% would work too, but 1 makes more sense to me for some reason.